cla_frontend/settings/base.py

```diff
@@ -115,7 +115,15 @@ def get_env_value(var_name):
 ANALYTICS_ID = os.environ.get("GA_ID", "")
 ANALYTICS_DOMAIN = os.environ.get("GA_DOMAIN", "")
 
-CSP_DEFAULT_SRC = ["'self'", "o345774.ingest.sentry.io", "ws:", "wss:", "www.google-analytics.com"]
+CSP_DEFAULT_SRC = [
+    "'self'",
+    "localhost:*",
```

Comment:
Is this to make local development easier? If so, is it possible to move it to local.example.py

Reply:
Hmm it might have been. Good point.

Reply:
local.py won't work for dockerised running (eg local k8s) but I'll work something out

Reply:
Modified it to key off ALLOWED_HOSTS. What do you reckon?
Tried to add the contents of ALLOWED_HOSTS directly, but had to do the `:*` bit so settled on this